FurCoNZ/ViewComponents/StripeViewComponent.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+using FurCoNZ.Configuration;
+using FurCoNZ.Models;
+using FurCoNZ.Services;
+using FurCoNZ.Services.Payment;
+using FurCoNZ.ViewModels;
+
+namespace FurCoNZ.Components
+{
+    public class StripeViewComponent : ViewComponent
+    {
+        private readonly IUserService _userService;
+        private readonly IOrderService _orderService;
+        private readonly StripeService _stripeService;
+        private readonly IOptions<StripeSettings> _stripeOptions;
+
+        public StripeViewComponent(IUserService userService, IOrderService orderService, StripeService stripeService, IOptions<StripeSettings> stripeOptions)
+        {
+            _userService = userService;
+            _orderService = orderService;
+            _stripeService = stripeService;
+            _stripeOptions = stripeOptions;
```

Comment:
Convention here is usually to call `.Value` on any injected `IOptions<T>` parameters
```suggestion
            _stripeOptions = stripeOptions.Value;
```

Reply:
💩